src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/CreateAccessRequirementButton.java

```diff
@@ -101,13 +82,72 @@ private void showIfACTMember() {
       new CallbackP<Boolean>() {
         @Override
         public void invoke(Boolean isACTMember) {
-          button.setVisible(isACTMember);
+          view.setButtonVisible(isACTMember);
         }
       }
     );
   }
 
+  @Override
+  public void onClick() {
+    if (DisplayUtils.isInTestWebsite(cookies)) {
+      useSrcWizard();
+    } else {
+      useSwcWizard();
+    }
+  }
+
+  private void useSwcWizard() {
+    CreateAccessRequirementWizard wizard =
+      ginInjector.getCreateAccessRequirementWizard();
```

Comment:
Maybe rename CreateAccessRequirementWizard to something like LegacyCreateAccessRequirementWizard for clarity when browsing the codebase?  (and ginInjector.getLegacyCreateAccess...)